lib/DOMMatrix.js

```diff
@@ -187,31 +187,189 @@ function setNumber3D(receiver, index, value) {
+    }
+  },
+  m13: {
+    get: function () {
+      return this._values[M13]
+    },
+    set: function (v) {
+      return setNumber3D(this, M13, v)
+    }
+  },
+  m14: {
+    get: function () {
+      return this._values[M14]
+    },
+    set: function (v) {
+      return setNumber3D(this, M14, v)
+    }
+  },
+  m21: {
+    get: function () {
+      return this._values[M21]
+    },
+    set: function (v) {
+      return setNumber2D(this, M21, v)
+    }
+  },
+  m22: {
+    get: function () {
+      return this._values[M22]
+    },
+    set: function (v) {
+      return setNumber2D(this, M22, v)
+    }
+  },
+  m23: {
+    get: function () {
+      return this._values[M23]
+    },
+    set: function (v) {
+      return setNumber3D(this, M23, v)
+    }
+  },
+  m24: {
+    get: function () {
+      return this._values[M24]
+    },
+    set: function (v) {
+      return setNumber3D(this, M24, v)
+    }
+  },
+  m31: {
+    get: function () {
+      return this._values[M31]
+    },
+    set: function (v) {
+      return setNumber3D(this, M31, v)
+    }
+  },
+  m32: {
+    get: function () {
+      return this._values[M32]
+    },
+    set: function (v) {
+      return setNumber3D(this, M32, v)
+    }
+  },
+  m33: {
+    get: function () {
+      return this._values[M33]
+    },
+    set: function (v) {
+      return setNumber3D(this, M33, v)
+    }
+  },
+  m34: {
+    get: function () {
+      return this._values[M34]
+    },
+    set: function (v) {
+      return setNumber3D(this, M34, v)
+    }
+  },
+  m41: {
+    get: function () {
+      return this._values[M41]
+    },
+    set: function (v) {
+      return setNumber2D(this, M41, v)
+    }
+  },
+  m42: {
+    get: function () {
+      return this._values[M42]
+    },
+    set: function (v) {
+      return setNumber2D(this, M42, v)
+    }
+  },
+  m43: {
+    get: function () {
+      return this._values[M43]
+    },
+    set: function (v) {
+      return setNumber3D(this, M43, v)
+    }
+  },
+  m44: {
+    get: function () {
+      return this._values[M44]
+    },
+    set: function (v) {
+      return setNumber3D(this, M44, v)
+    }
+  },
+
+  a: {
+    get: function () {
+      return this.m11
+    },
+    set: function (v) {
+      return this.m11 = v
+    }
+  },
+  b: {
+    get: function () {
+      return this.m12
+    },
+    set: function (v) {
+      return this.m12 = v
+    }
+  },
+  c: {
+    get: function () {
+      return this.m21
+    },
+    set: function (v) {
+      return this.m21 = v
+    }
+  },
+  d: {
+    get: function () {
+      return this.m22
+    },
+    set: function (v) {
+      return this.m22 = v
+    }
+  },
+  e: {
+    get: function () {
+      return this.m41
+    },
+    set: function (v) {
+      return this.m41 = v
+    }
+  },
+  f: {
+    get: function () {
+      return this.m42
+    },
+    set: function (v) {
+      return this.m42 = v
+    }
+  },
+
+  is2D: {
+    get: function () {
+      return this._is2D
+    }
+  }, // read-only
```

Comment:
Hmm, I think that this was much more readable before? 🤔 

I don't think that there is anything in `standard` that prevents us from still having this be one-liners?